games-roguelike/nethack/nethack-3.6.0.recipe

```diff
@@ -0,0 +1,85 @@
+SUMMARY="The ultimate old-school single player dungeon exploration game"
+HOMEPAGE="http://www.nethack.org/"
+DESCRIPTION="NetHack is a single-player roguelike video game originally \
+released in 1987 with ASCII graphics. It is a descendant of an earlier game \
+called Hack (1985), which is a descendant of Rogue (1980)."
+COPYRIGHT="1985-2015 Stichting Mathematisch Centrum and M. Stephenson"
+REVISION="1"
+LICENSE="nethack"
+SOURCE_URI="http://sourceforge.net/projects/nethack/files/nethack/3.6.0/nethack-360-src.tgz"
+CHECKSUM_SHA256="1ade698d8458b8d87a4721444cb73f178c74ed1b6fde537c12000f8edf2cb18a"
+SOURCE_DIR="nethack-$portVersion"
+
+PATCHES="nethack-$portVersion.patchset"
+
+#the x86 is left is somebody wanted to edit it to work under gcc4
+#(it compiles in gcc4, but what is the point if it compiles under gcc2)
+ARCHITECTURES="x86_gcc2 !x86 ?x86_64"
```

Comment:
If it compiles in GCC2 *and* GCC4, then get rid of SECONDARY_ARCHITECTURES (+ all `$secondaryArchSuffix`es), and change the `!x86` to `?x86`.